server/repository/migrations/postgres/2023-03-05-205307_create_name_trigger/up.sql

```diff
@@ -0,0 +1,29 @@
+ALTER TYPE changelog_table_name ADD VALUE 'name' AFTER 'clinician_store_join';
```

Comment:
the value `name` already exists, having it here will cause the server to panic!
( is in `22022-01-27T08-00_create_changelog_table/up.sql` )

You can add `if not exists` but better to just remove I think

Reply:
[7f52ffe](https://github.com/openmsupply/open-msupply/pull/1278/commits/7f52ffee2d42fbc9c42921dfd9066bc0f51cbdd8)~